american-truck-simulator.kvp

```diff
@@ -0,0 +1,98 @@
+Meta.DisplayName=American Truck Simulator
+Meta.Description=American Truck Simulator Dedicated Server
+Meta.OS=Windows, Linux
+Meta.AarchSupport=NotSupported
+Meta.Arch=x86_64
+Meta.Author=IceOfWraith
+Meta.URL=https://store.steampowered.com/app/270880/American_Truck_Simulator/
+Meta.DisplayImageSource=steam:270880
+Meta.EndpointURIFormat=steam://connect/{ip}:{GenericModule.App.Ports.$SteamQueryPort}
+Meta.ConfigManifest=american-truck-simulatorconfig.json
+Meta.MetaConfigManifest=american-truck-simulatormetaconfig.json
+Meta.ConfigRoot=american-truck-simulator.kvp
+Meta.MinAMPVersion=2.4.3.0
+Meta.SpecificDockerImage=
+Meta.DockerRequired=False
+Meta.ContainerPolicy=SupportedOnLinux, RequiredOnWindows
+Meta.ContainerPolicyReason=The game server does not run properly as NETWORK SERVICE, so it must be run within Docker on Windows.
+Meta.ExtraSetupStepsURI=https://discourse.cubecoders.com/docs?topic=3790?utm_source=ampcreate&utm_content=ats-ets2
+Meta.Prerequsites=[]
+Meta.ExtraContainerPackages=[]
+Meta.ConfigReleaseState=NotSpecified
+Meta.NoCommercialUsage=False
+App.DisplayName=American Truck Simulator
+App.RootDir=./american-truck-simulator/
+App.BaseDirectory=./american-truck-simulator/2239530/
+App.SteamWorkshopDownloadLocation=
+App.ExecutableWin=2239530\bin\win_x64\amtrucks_server.exe
```

Comment:
Should this be included if it can't be run natively?

Reply:
Normally, I'd keep it included for the eventual hope that AMP would support it properly. But you're right, it would cause issues if someone create it without Docker installed which is totally doable right now.